docker-compose.yml

```diff
@@ -4,12 +4,8 @@
 
 version: '2'
 services:
-  bmo:
+  bmo.dev:
     image: mozillaconduit/docker-bmo
-    ports:
-      - "7777:80"
-    volumes:
-      - bmo-mysql-db:/var/lib/mysql
```

Comment:
This was discussed with dkl in irc, the docker-bmo repo needs some work and something like this will be added back in later.